data_steward/utils/participant_summary_requests.py

```diff
@@ -157,6 +172,73 @@ def get_deactivated_participants(project_id, dataset_id, tablename, columns):
     return df
 
 
+def get_site_participant_information(project_id, hpo_id):
+    """
+    Fetches the necessary participant information for a particular site.
+
+    :param project_id: The RDR project that contains participant summary data
+    :param hpo_id: awardee name of the site
+
+    :return: returns dataframe of participant information
+    """
+    # Parameter checks
+    if not isinstance(project_id, str):
+        raise RuntimeError(f'Please specify the RDR project')
+
+    if not isinstance(hpo_id, str):
+        raise RuntimeError(f'Please provide an hpo_id')
+
+    token = get_access_token()
+
+    headers = {
+        'content-type': 'application/json',
+        'Authorization': 'Bearer {0}'.format(token)
```

Comment:
Could use an f-string for this formatting.
```suggestion
        'Authorization': f'Bearer {token}'
```